src/explore-education-statistics-common/src/components/form/FormCheckboxSearchSubGroups.tsx

```diff
@@ -187,17 +198,19 @@ const FormCheckboxSearchSubGroups = ({
                 options={optionGroup.options}
                 value={value}
                 selectAll
-                selectAllText={(allChecked, opts) =>
-                  `${allChecked ? 'Unselect' : 'Select'} all ${
-                    opts.length
-                  } subgroup options`
-                }
+                selectAllText={(allChecked, opts) => (
+                  <>
+                    {`${allChecked ? 'Unselect' : 'Select'} all ${
+                      opts.length
+                    } subgroup options`}
```

Comment:
Similar to my other comment about pluralisation

Reply:
As above, the button only shows if there's more than one option.